src/commands/party.js

```diff
@@ -1,8 +1,27 @@
+import axios from 'axios';
+import Discord from 'discord.js';
 import DiscordCommand from './command.js';
 
 export default class PartyCommand extends DiscordCommand {
-    async execute() {
-        const party = this.getEmojiByName('PepeHypeDance');
-        this.message.channel.send(`${party}${party}${party}`);
+  async execute() {
+    if (!this.member) {
+      this.send("You must be in a Roo Bot enabled Discord server to use this command");
+      return;
     }
+
+    const party = this.getEmojiByName('PepeHypeDance');
+    this.message.react(party);
+
+    axios.get(
```

Comment:
use `const response = await axios.get(...)` to unnest a bit here and we will get better errors too